static_src/components/app_list.jsx

```diff
@@ -69,13 +69,13 @@ export default class AppList extends React.Component {
     var content = <h4 className="test-none_message">No apps</h4>;
     // TODO format rows in table
     if (this.state.apps.length) {
-      content = <Table data={ this.getRows(this.state.apps) } 
+      content = <Table data={ this.getRows(this.state.apps) }
         columns={ this.columns }
-        sortable={ true } className="table" />;
+        sortable={ true } />;
```

Comment:
I unfortunately *may* have used ".table" in the acceptance tests. They also seem to be failing on travis, although that may be some unrelated error (I tried re-running them). Did they run successfully locally?